core/state_processor_test.go

```diff
@@ -528,3 +531,42 @@ func TestProcessVerkle(t *testing.T) {
 		}
 	}
 }
```

Comment:
It would be nice to also read the verkle tree to check that the block hashes are there, so that we can make sure that if the stuff changes in the future, we don't forget to test that.

Reply:
Sure, added verkle to the test.